panel/src/config/fiber.js

```diff
@@ -260,6 +260,7 @@ const Fiber = {
     {
       data = {},
       headers = {},
+      include = [],
```

Comment:
We now have `only` and `include` - to me that totally sound the same. What's the difference? Or can we merge them?

Reply:
Here's the explanation: 

only: is needed in cases when you want to reload a single part of the view response (i.e. the lock info) You send the fields that you want to get back and they response is merged with what's already there. 

include: is needed when you want to include globals in the view response. I.e. to update the $translation. In this case you get a full view response back, but you also include a global key that would normally not be included. I'm definitely open for better names. I liked "include" as we use it in a similar way for API calls.

Reply:
Ok I understand it now, but I'd say we definitely need different names. Cause implicitly they both are saying "only include"/"include only". Why don't we just call them `props` and `globals`. If you don't specify any, you get all. As soon as you specify them, you only get these. And by the name, you already know what this refers to.

Reply:
I like that a lot.

Reply:
After rethinking it some more, I think it's wrong to split it into props and globals. In practice, it's more about the task. 

You want to load `only` one item or you want to `replace` items in the response/state.
Items can either be globals or props. 

That's why I renamed the include parameter into replace. I think this makes it a lot clearer what's the expected outcome.

I.e.

```js
this.$reload({
  replace: '$translation'
});
```

vs. 

```js
this.$reload({
  only: '$view.props.lock'
});
```

In general, replace is the way to go. It will refresh the rest of the view state as well and state changes will always be complete. 

Only is "only" needed for stuff like content locking when you really just need that one piece of the state to be refreshed.

Reply:
Damn, directly after writing the comment I'm not so sure again about `replace`. The intention of the call is to reload the view, but also include a global. So it should actually probably be: 

```js
this.$reload({
  globals: '$translation'
});
```

vs. 

```js
this.$reload({
  only: '$view.props.lock'
});
```
 
The first one will return the entire view response + the requested globals. 
The second one will only return a particular prop. That can be a shared prop or a global prop. I.e. this would also work

```js
this.$reload({
  only: '$translation'
});
```

But in this case the response will only contain the $translation and nothing else.

Reply:
"globals" and "only" it is!